server/src/main/java/com/distkv/server/Location.java

```diff
@@ -0,0 +1,35 @@
+package com.distkv.server;
+
+/**
+ * the location of node.
+ */
+public class Location {
```

Comment:
Is this class necessary?

Reply:
It represent the location of nodes.  If the nodes at same rack, the network delay is low, but the probability of failure together is high. on the another hand, the nodes at different racks has higher network delay, but with lower probability of failure.  for data with there replication, one replication should at different rack.

Reply:
I think this function may be considered after our cluster mode is implemented. What do you think?